### DISCUSSION TITLE
Adding duplicate instance/ database names of different domains

### DISCUSSION THREAD

**Discussion by NicoBeekelaar** [trimble-oss/dba-dash#867] [General]
Hi, 
I'm trying to add SQL Server instances from different domains. That's totally ok if the names are different, but if they're identical it looks like DBA Dash merges the data of the 2 instances. There isn't any new row in the Instances table, so it looks like the instance name should be unique. The ip addresses of the servers are different. Is there any smart way to combine these instances in 1 repository (using aliases or something)? If not, I have to install a separate service and repository database for these duplicate-name instances.
Thnx, Nico

**Comment by DavidWiseman** [comment]
Hi, in the service configuration tool on the **Source** tab, specify a **ConnectionID**.  This is what's used to uniquely identify an instance.  If it's not set, the instance name is used.  You just need to make sure the value you use is unique across the domains.  
The ConnectionID will also be used as the name of the instance in the GUI unless an alias is specified.

Hope this helps

**Reply by NicoBeekelaar** [reply]
Thnx David, works like a charm!
